### PR TITLE
Display help if no args are passed

### LINKED ISSUE #64: Add -h and --help options, printing short usage info
Also, when running jet without any arguments, it should perhaps behave as `jet -h`?

Thanks a lot for the tool!

### PR DESCRIPTION
**Pull request by kolharsam**
This should resolve #64

### CODE REVIEW

File: src/jet/main.clj [hunk 1]
```diff
@@ -94,7 +94,9 @@
                 :interactive :collect
                 :edn-reader-opts
                 :help]} (parse-opts args)]
-    (cond version
+    (if (nil? args)
+      (print-help)
+      ((cond version
```
Comment (borkdude) [inline src/jet/main.clj:99]: This is a typo probably. ((cond ...) should be (cond ...)

Comment (kolharsam) [reply src/jet/main.clj:99]: Also, I noticed that one of the tests, the `interactive-test` had the no-args as a test case. Would we have to make a change to that?

**Comment by kolharsam** [comment]
Now the `stream-test` is failing. 🤦‍♂️
I'll look into that.

**Comment by kolharsam** [comment]
> Now the `stream-test` is failing. 🤦‍♂️
> I'll look into that.

To fix this, do we add another argument for streaming?

File: src/jet/main.clj [hunk 2]
```diff
@@ -94,7 +94,9 @@
                 :interactive :collect
                 :edn-reader-opts
                 :help]} (parse-opts args)]
-    (cond version
+    (if (nil? args)
```
Comment (borkdude) [inline src/jet/main.clj:97]: I think `(nil? args)` could be moved into the `cond`

Comment (kolharsam) [reply src/jet/main.clj:97]: Right, yes! That would work!

Comment (kolharsam) [reply src/jet/main.clj]: Changed it!

**Comment by borkdude** [comment]
Thanks.

**Pull request merged**